server/persistence/mongodb/mongodb_service_game.go

```diff
@@ -243,6 +252,29 @@ func validateGameState(savedGames []model.Game, newGameState model.Game) error {
 	return nil
 }
 
+func validateLatestActionBelongs(mg model.Game) error {
```

Comment:
This looks verbatim to the one in `memory_service_game.go`, let's make it a util function somewhere instead